src/main/resources/schema.sql

```diff
@@ -128,6 +128,7 @@ CREATE TABLE `educational_group` (
   `status_id` INT NOT NULL,
   `specialization_id` INT NOT NULL,
   `budget_owner_id` INT NOT NULL,
+  `is_deleted` tinyint(1) NOT NULL,
```

Comment:
Its not true. Change data.sql insert to this for example
`insert into educational_group (`name`,location_id,start_date,finish_date,status_id,specialization_id,budget_owner_id) values ('DP-116',1,'2017-04-29', '2017-08-18',6,6,1 ); ` 
And you will get `SQLException: Field 'is_deleted' doesn't have a default value`